collection_processing.py

```diff
@@ -20,32 +20,56 @@ def tokenize_simple(text):
         return tokens
 
 
+def article_tokenize_other(text):
```

Comment:
Could you give it a more explicit name ?
For example `tokenize_regex` (you can remove the `article` prefix).